tests/foreman/cli/test_errata.py

```diff
@@ -372,25 +374,7 @@ def test_errata_list_1(self):
         """
 
     @stubbed()
-    def test_errata_list_2(self):
```

Comment:
Yes, this is a dupe and covered by another test `test_positive_list_filter_by_product_name`